Rack/rack_app.rb

```diff
@@ -0,0 +1,53 @@
+#frozen_string_literal: true
+
+require './config/initializers/initializer'
+require './run.rb'
+
+class RackApp
+  def call(env)
+    request = Rack::Request.new(env)
+
+    case request.path.start_with?
+    when '/api/sups'
+      Handlers::Sup.new(request).response
+    when 'api/customers'
+      Handlers::Customer.new(request).response
+    else
+      Rack::Base.response_404
```

Comment:
I meant you should use `Handlers::Base.new.response_404`.
Cause `Rack::Base` will return you `uninitialized constant Rack::Base`

Reply:
done